node/pkg/adminrpc/adminserver.go

```diff
@@ -527,7 +596,10 @@ func ibcUpdateChannelChain(
 	if len(req.ChannelId) > 64 {
 		return nil, fmt.Errorf("invalid channel ID length, must be <= 64")
 	}
-	channelId := vaa.LeftPadIbcChannelId(req.ChannelId)
+	channelId, err := vaa.LeftPadIbcChannelId(req.ChannelId)
```

Comment:
How come you did the padding out here when some of the others do the padding inside the .serialize? Like payloads.go::vaa.BodyRecoverChainId

Reply:
Well, I didn't  do this, it was existing code. But I think the reason is because the channel ID passed into the serialize method is a byte array, not a string.